application/templates/grafanadashboard.yaml

```diff
@@ -0,0 +1,28 @@
+
+{{- if .Values.grafanaDashboard -}}
+{{- if .Values.grafanaDashboard.enabled -}}
+{{- range $name, $content := .Values.grafanaDashboard.contents }}
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDashboard
+metadata:
+  labels:
+    grafanaDashboard: grafana-operator
+{{ include "application.labels.stakater" $ | indent 4 }}
+{{ include "application.labels.chart" $ | indent 4 }}
+{{- if $.Values.grafanaDashboard.additionalLabels }}
+{{ toYaml $.Values.grafanaDashboard.additionalLabels | indent 4 }}
+{{- end }}
```

Comment:
lets add support for annotations as well

Reply:
Will add it in next commit